pom.xml

```diff
@@ -11,10 +11,10 @@
     <url>https://github.com/latera/camunda-ext</url>
   </scm>
   <properties>
-    <kotlin.version>1.4.21</kotlin.version>
+    <kotlin.version>1.4.30-RC</kotlin.version>
```

Comment:
Nope, i think we should not use RC version

Reply:
Ok, the last one is [1.4.21](https://kotlinlang.org/releases.html), so we don't need to update it.